src/Pyz/Client/Training/TrainingClient.php

```diff
@@ -0,0 +1,113 @@
+<?php declare(strict_types = 1);
+
+namespace Pyz\Client\Training;
+
+use Generated\Shared\Transfer\ProductViewTransfer;
+use Generated\Shared\Transfer\TrainingStorageItemTransfer;
+use Pyz\Shared\Training\TrainingConstants;
+use Spryker\Client\Kernel\AbstractClient;
+
+/**
+ * @method \Pyz\Client\Training\TrainingFactory getFactory()
+ */
+class TrainingClient extends AbstractClient implements TrainingClientInterface
```

Comment:
Client is like the business facade in Yves/Zed. Please extract all logic to a class, maybe write a `Collector` class to do this aggregation and moving around. That way you have a clean Client 😉